app/src/main/java/edu/uc/mille5bm/smartnutrition/service/FoodService.kt

```diff
@@ -9,8 +9,9 @@ import retrofit2.Callback
 import retrofit2.Response
 
 class FoodService {
-    fun fetchFoods() : MutableLiveData<ArrayList<Food>> {
-        var _foods = MutableLiveData<ArrayList<Food>>()
+    fun fetchFoods(foodName: String): MutableLiveData<ArrayList<Food>> {
+        //local variable should start with lowercase letter
```

Comment:
Putting conversation/discussion  in code comments is bad practice, as that clutters up the source code and does not belong in source code files.  I mentioned that in the code review instructions.    Discussion and comments are very good, but have that in the appropriate place, which is in the "Files Changed" tab in the pull request in GitHub (which is where I am writing this and you are reading this).  That way, you can have a conversation with replies, right where the code is happening, and without cluttering up the source code itself.